src/core/mac/mac.cpp

```diff
@@ -435,6 +435,15 @@ void Mac::SetSupportedChannelMask(const ChannelMask &aMask)
     return;
 }
 
+uint32_t Mac::GetPhySupportedChannelMask(void) const
+{
+#if OPENTHREAD_CONFIG_RADIO_CHANNEL_MASK
```

Comment:
Is it possible to provide a weak implementation for so `otPlatRadioGetSupportedChannelMask()` that we don't need this macro.

Reply:
This macro will also help to reduce code size when Radio channel mask related features are not enabled.